docs/language/operators/pass.md

```diff
@@ -0,0 +1,26 @@
+### Operator
+
+&emsp; **pass** &mdash; copy input values to output
+
+### Synopsis
+
+```
+pass
+```
+### Description
+
+The `pass` operator outputs a copy of each input value. It is typically used
+with operators that handle multiple legs of the dataflow path such as
+[`fork`](fork.md) and [`join`](join.md).
+
+### Examples
+
+_Copy input to output_
+```mdtest-command
+echo -e "hello\nworld" | zq -z -i line 'pass' -
```

Comment:
@nwt: Got it... simple is good. If it won't offend, though, I'm inclined to keep the single quotes around `'pass'`. I know strictly speaking they're unnecessary in this case, but since almost all programs have something in them that require they be quoted, I'm in favor of keeping the docs examples consistent with that.

Reply:
Please drop the quotes, both because we quote single-word Zed programs almost nowhere else and because useless quoting drives me nuts.